### ISSUE TITLE
NemeStats doesn't handle games where the lowest score is the winner

### ISSUE THREAD

**Issue by HolisticDeveloper** [NemeStats/NemeStats#671]
My group played No Thanks! tonight where the person with the lowest score is the winner. There doesn't seem to be a way to enter the score and have NemeStats sort in reverse order. Does this need to be entered as a ranked game instead?

**Comment by jakejgordon** [comment]
You are correct. We should address this, but entering as a ranked game is the workaround.
